x-pack/plugin/blob-cache/src/test/java/org/elasticsearch/blobcache/shared/SharedBlobCacheServiceTests.java

```diff
@@ -72,6 +73,13 @@ private static long size(long numPages) {
         return numPages * SharedBytes.PAGE_SIZE;
     }
 
+    private static <E extends Exception> void completeWith(ActionListener<Void> listener, CheckedRunnable<E> runnable) {
```

Comment:
nit Isn't this equivalent to runBefore?

Reply:
It's equivalent of `runBefore().onResponse(null)` or `ActionRunnable.run(listener, runnable).run()`, but seem more clearer.